modules/persistenceService/src/main/resources/db/migration/V0__CREATE_USER_TABLE.sql

```diff
@@ -0,0 +1,4 @@
+create table user(
+    id serial PRIMARY KEY,
+    username VARCHAR UNIQUE NOT NULL
```

Comment:
Should we add a new column for the name?

Reply:
I don't think I got this one if you could please elaborate.